src/main.js

```diff
@@ -91,10 +91,32 @@ function createModulesTableCell(text) {
 /**
  * Create one row of the modules table.
  */
-function createModulesTableRow(parsedModule) {
+async function createModulesTableRow(parsedModule) {
```

Comment:
I you wouldn't load the JSON for every call of `i18n.getMessage()`, this method wouldn't need to be async.

Reply:
done